app/pages/admin/organization-status-list.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import LoadingIndicator from '../../components/loading-indicator';
+import ProjectIcon from '../../components/project-icon';
+import Paginator from '../../talk/lib/paginator';
+
+class OrganizationStatusList extends Component {
+  constructor(props) {
+    super(props);
+    this.getOrganizations = this.getOrganizations.bind(this);
+    this.renderOrganizationList = this.renderOrganizationList.bind(this);
+    this.renderOrganizationListItem = this.renderOrganizationListItem.bind(this);
+    this.state = {
+      loading: false,
+      organizations: [],
+      error: null
+    };
+  }
+
+  componentDidMount() {
+    this.getOrganizations();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.location.query !== this.props.location.query) {
+      this.getOrganizations();
+    }
+  }
+
+  getOrganizations() {
+    const { query } = this.props.location;
+
+    const organizationsQuery = {
+      include: 'avatar'
```

Comment:
There's a bug with the json-api-client that if you are including a single resource that it must be in an array: https://github.com/zooniverse/json-api-client/issues/35